config/workers.coffee

```diff
@@ -8,6 +8,11 @@ module.exports = (KONFIG, options, credentials) ->
   nodeProgram = if options.watchNode then './watch-node' else 'node'
 
   workers =
+    nginx:
+      group: 'webserver'
+      supervisord:
+        command: 'nginx -c %(ENV_KONFIG_PROJECTROOT)s/nginx.conf'
```

Comment:
could you also change the "stopsignal" to QUIT for supervisord conf of this worker? we will make sure that nginx is restarted/stopped [gracefully](http://github.perusio.org/nginx-manpage/#SIGNALS).